### ISSUE TITLE
JAWS And Chrome RadioGroups using aria-activedescendant RadioGroup Name not conveyed when speaking Window Prompt and Text with JAWSKEY+TAB

### ISSUE THREAD

**Issue by jhartleytpgi** [FreedomScientific/VFO-standards-support#854]
### Summary
When a radio button is navigated to , and JAWSKEY+TAB is used to speak the window prompt and text with either the PC or the virtual PC cursor active, the radio button group name is not announced along with the selected radio button. For Example, JAWS announces "Regular crust radio button checked", without announcing the radio button group name "Pizza Crust group."

### Expected result
Expected behavior is JAWS to announce the name of the group along with the radio button currently selected. Example:
"Pizza Crust group, Regular crust radio button checked". 
### Actual result

JAWS only announces "Regular crust radio button checked", without announcing the group name.

### Example
https://aria-at.netlify.app/tests/radiogroup-aria-activedescendant/reference/2022-4-7_113015/radio-activedescendant.setfocusonfirstradiobutton
#### JAWS version and build number
2024.2406.121

#### Operating System and version
Windows 10 / Windows 11

#### Browser and version:
Google Chrome
126.0.6478.183 (Official Build) (64-bit)

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 92813
